plugins/xpad/xpad.cpp

```diff
@@ -23,6 +23,10 @@
 #include "xpad.h"
 #include <VersionHelpers.h>
 
+#undef _WIN32_WINNT
+#define _WIN32_WINNT 0x0602 // Required for XInputEnable definition
```

Comment:
I fear we are talking of [two](http://pastebin.com/hvdxwTYm) completely [different](http://pastebin.com/WtqThzSp) files.